bin/polyserve

```diff
@@ -24,6 +24,7 @@ Options:
   -b <browsername> use this browser instead of default
                    (ex: 'Google Chrome Canary')
   -H <hostname>    use this hostname instead of localhost
+  --cors           enable permissive cors headers in server responses
```

Comment:
I would consider require the `Access-Control-Allow-Origin` value to be passed in via the flag. That way there's some control over the setting and the permissive mode requires `polyserve --cors=*`